### PR TITLE
[FLINK-18973][docs-zh] Translate the 'History Server' page of 'Debugging & Monitoring' into Chinese

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

*Translate the 'History Server' page of 'Debugging & Monitoring' into Chinese*

## Brief change log

*Translate the 'History Server' page of 'Debugging & Monitoring' into Chinese*
  - The page url is https://ci.apache.org/projects/flink/flink-docs-release-1.11/zh/monitoring/historyserver.html
  - The markdown file is located in flink/docs/monitoring/historyserver.zh.md

## Verifying this change

*This change is a pure documentation translation.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? no

### CODE REVIEW

File: docs/monitoring/historyserver.zh.md [hunk 1]
```diff
@@ -22,62 +22,69 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-Flink has a history server that can be used to query the statistics of completed jobs after the corresponding Flink cluster has been shut down.
+Flink 提供了 history server，可以在相应的 Flink 集群关闭之后查询已完成作业的统计信息。
 
-Furthermore, it exposes a REST API that accepts HTTP requests and responds with JSON data.
+此外，它暴露了一套 REST API，该 API 接受 HTTP 请求并以 JSON 数据进行响应。
```
Comment (RocMarshal) [inline docs/monitoring/historyserver.zh.md:27]:
```suggestion
此外，它暴露了一套 REST API，该 API 接受 HTTP 请求并以 JSON 数据格式进行响应。
```

**Comment by RocMarshal** [comment]
Hi @klion26 
Could you help me to review this PR if you have free time?
Thank you.

File: docs/monitoring/historyserver.zh.md [hunk 2]
```diff
@@ -22,62 +22,67 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-Flink has a history server that can be used to query the statistics of completed jobs after the corresponding Flink cluster has been shut down.
+Flink 提供了 history server，可以在相应的 Flink 集群关闭之后查询已完成作业的统计信息。
 
-Furthermore, it exposes a REST API that accepts HTTP requests and responds with JSON data.
+此外，它暴露了一套 REST API，该 API 接受 HTTP 请求并以 JSON 数据格式进行响应。
```
Comment (klion26) [inline docs/monitoring/historyserver.zh.md:27]: `并返回 JSON 格式的数据` 会好一些吗？

File: docs/monitoring/historyserver.zh.md [hunk 3]
```diff
@@ -22,62 +22,67 @@ specific language governing permissions and limitations
...
-## Overview
+<a name="overview"></a>
 
-The HistoryServer allows you to query the status and statistics of completed jobs that have been archived by a JobManager.
+## 概览
 
-After you have configured the HistoryServer *and* JobManager, you start and stop the HistoryServer via its corresponding startup script:
+HistoryServer 允许查询 JobManager 存档的已完成作业的状态和统计信息。
+
+在配置 HistoryServer *和* JobManager 之后，你可以使用其相应的启动脚本来启动和停止 HistoryServer：
```
Comment (klion26) [inline docs/monitoring/historyserver.zh.md:38]: `你可以使用相应的脚本来启动和停止 HistoryServer` 会好一些吗？

File: docs/monitoring/historyserver.zh.md [hunk 4]
```diff
@@ -22,62 +22,67 @@ specific language governing permissions and limitations
...
 
 {% highlight shell %}
-# Start or stop the HistoryServer
+# 启动或者停止 HistoryServer
 bin/historyserver.sh (start|start-foreground|stop)
 {% endhighlight %}
 
-By default, this server binds to `localhost` and listens at port `8082`.
+默认情况下，此服务器绑定到 `localhost` 并监听 `8082` 端口。
```
Comment (klion26) [inline docs/monitoring/historyserver.zh.md:45]: “此服务器绑定到 `localhost` 的 `8082` 端口”  这样描述会好一些吗？

File: docs/monitoring/historyserver.zh.md [hunk 5]
```diff
@@ -22,62 +22,67 @@ specific language governing permissions and limitations
...
+目前，只能将 HistoryServer 作为独立的进程运行。
 
-Currently, you can only run it as a standalone process.
+<a name="configuration"></a>
 
-## Configuration
+## 配置参数
 
-The configuration keys `jobmanager.archive.fs.dir` and `historyserver.archive.fs.refresh-interval` need to be adjusted for archiving and displaying archived jobs.
+配置项 `jobmanager.archive.fs.dir` 和 `historyserver.archive.fs.refresh-interval` 需要根据归档路径和已归档作业刷新间隔进行调整。
```
Comment (klion26) [inline docs/monitoring/historyserver.zh.md:53]: `和已归档作业刷新间隔进行调整` 这里的描述可以看出意思，不过读起来有点点拗口，这里有更好的描述吗？或者看看其他人是否有更好的建议

File: docs/monitoring/historyserver.zh.md [hunk 6]
```diff
@@ -22,62 +22,67 @@ specific language governing permissions and limitations
...
 
 **JobManager**
 
-The archiving of completed jobs happens on the JobManager, which uploads the archived job information to a file system directory. You can configure the directory to archive completed jobs in `flink-conf.yaml` by setting a directory via `jobmanager.archive.fs.dir`.
+已完成作业的存档在 JobManager 上进行，将已存档的作业信息上传到文件系统目录中。你可以在 `flink-conf.yaml` 文件中通过 `jobmanager.archive.fs.dir` 设置一个目录存档已完成的作业。
```
Comment (klion26) [inline docs/monitoring/historyserver.zh.md:57]: 上面用 `归档`， 这里用了 `存档`，建议全文可以统一

File: docs/monitoring/historyserver.zh.md [hunk 7]
```diff
@@ -22,62 +22,67 @@ specific language governing permissions and limitations
...
 
-## Available Requests
+<a name="available-requests"></a>
 
-Below is a list of available requests, with a sample JSON response. All requests are of the sample form `http://hostname:8082/jobs`, below we list only the *path* part of the URLs.
+## 可用的请求
 
-Values in angle brackets are variables, for example `http://hostname:port/jobs/<jobid>/exceptions` will have to requested for example as `http://hostname:port/jobs/7684be6004e4e955c2a558a9bc463f65/exceptions`.
+以下是可用且带有示例 JSON 响应的请求列表。所有请求格式样例均为 `http://hostname:8082/jobs`，下面我们仅列出了 URLs 的 *path* 部分。
+尖括号中的值是变量，例如 `http://hostname:port/jobs/<jobid>/exceptions` 将必须请求如 `http://hostname:port/jobs/7684be6004e4e955c2a558a9bc463f65/exceptions`。
```
Comment (klion26) [inline docs/monitoring/historyserver.zh.md:85]: 这句话读起来有些拗口，主要是后面的 ”将必须请求如 xxx“ 有更好的描述吗？

**Review by klion26 — COMMENTED** [review]
@RocMarshal thanks for your contribution, left some comments. please let me know what do you think, thanks.

File: docs/monitoring/historyserver.zh.md (hunk 2 above)
Comment (RocMarshal) [inline docs/monitoring/historyserver.zh.md:27]:
```suggestion
此外，它暴露了一套 REST API，该 API 接受 HTTP 请求并返回 JSON 格式的数据。
```

File: docs/monitoring/historyserver.zh.md (hunk 3 above)
Comment (RocMarshal) [inline docs/monitoring/historyserver.zh.md:38]:
```suggestion
在配置 HistoryServer *和* JobManager 之后，你可以使用相应的脚本来启动和停止 HistoryServer：
```

File: docs/monitoring/historyserver.zh.md (hunk 4 above)
Comment (RocMarshal) [inline docs/monitoring/historyserver.zh.md:45]:
```suggestion
默认情况下，此服务器绑定到 `localhost` 的 `8082` 端口。
```

File: docs/monitoring/historyserver.zh.md (hunk 5 above)
Comment (RocMarshal) [inline docs/monitoring/historyserver.zh.md:53]:
```suggestion
配置项 `jobmanager.archive.fs.dir` 和 `historyserver.archive.fs.refresh-interval` 需要根据存档路径和已存档作业刷新间隔进行调整。
```

File: docs/monitoring/historyserver.zh.md [hunk 8]
```diff
@@ -22,62 +22,67 @@ specific language governing permissions and limitations
...
```
Comment (RocMarshal) [inline docs/monitoring/historyserver.zh.md:85]:
```suggestion
尖括号中的值为变量，例如作业 `7684be6004e4e955c2a558a9bc463f65` 的 
`http://hostname:port/jobs/<jobid>/exceptions` 请求须写为 `http://hostname:port/jobs/7684be6004e4e955c2a558a9bc463f65/exceptions`。
```

File: docs/monitoring/historyserver.zh.md [hunk 9]
```diff
@@ -22,62 +22,67 @@ specific language governing permissions and limitations
...
+目前，只能将 HistoryServer 作为独立的进程运行。
 
-Currently, you can only run it as a standalone process.
+<a name="configuration"></a>
 
-## Configuration
+## 配置参数
 
-The configuration keys `jobmanager.archive.fs.dir` and `historyserver.archive.fs.refresh-interval` need to be adjusted for archiving and displaying archived jobs.
+配置项 `jobmanager.archive.fs.dir` 和 `historyserver.archive.fs.refresh-interval` 需要根据存档路径和已存档作业刷新间隔进行调整。
```
Comment (RocMarshal) [inline docs/monitoring/historyserver.zh.md:53]:
```suggestion
配置项 `jobmanager.archive.fs.dir` 和 `historyserver.archive.fs.refresh-interval` 需要根据`作业存档目录`和`刷新作业存档目录的时间间隔`进行调整。
```

File: docs/monitoring/historyserver.zh.md [hunk 10]
```diff
@@ -22,62 +22,68 @@ specific language governing permissions and limitations
...
+目前，只能将 HistoryServer 作为独立的进程运行。
 
-Currently, you can only run it as a standalone process.
+<a name="configuration"></a>
 
-## Configuration
+## 配置参数
 
-The configuration keys `jobmanager.archive.fs.dir` and `historyserver.archive.fs.refresh-interval` need to be adjusted for archiving and displaying archived jobs.
+配置项 `jobmanager.archive.fs.dir` 和 `historyserver.archive.fs.refresh-interval` 需要根据`作业存档目录`和`刷新作业存档目录的时间间隔`进行调整。
```
Comment (RocMarshal) [inline docs/monitoring/historyserver.zh.md:53]:
```suggestion
配置项 `jobmanager.archive.fs.dir` 和 `historyserver.archive.fs.refresh-interval` 需要根据 `作业存档目录` 和 `刷新作业存档目录的时间间隔` 进行调整。
```

**Comment by RocMarshal** [comment]
@klion26 
Thank you very much for your guidance and advices. I made some adjustments according to your suggestions.
If you have time, please have a look. Looking forward to your further suggestions.

**Review by klion26 — APPROVED** [review]
@RocMarshal thanks for addressing the comments. LGTM now
ping @wuchong @rmetzger

**Comment by wuchong** [comment]
Merging...

**Pull request merged**